lib/modules/manager/mix/artifacts.ts

```diff
@@ -14,6 +14,10 @@ import * as hostRules from '../../../util/host-rules';
 import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
 
 const hexRepoUrl = 'https://hex.pm/';
+const hexRepoOrgUrlRegex = new RegExp(
```

Comment:
```suggestion
const hexRepoOrgUrlRegex = regEx(
```
use our regex helper function